test/acceptance/integration/IOTCORE-124 ServicePath for registrations (NGSI9) (not recursive)/iotcore-124.feature

```diff
@@ -0,0 +1,450 @@
+# -*- coding: utf-8 -*-
+"""
+Copyright 2014 Telefonica Investigación y Desarrollo, S.A.U
+
+This file is part of fiware-orion
+
+fiware-orion is free software: you can redistribute it and/or
+modify it under the terms of the GNU Affero General Public License as
+published by the Free Software Foundation, either version 3 of the License,
+or (at your option) any later version.
+
+fiware-orion is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+See the GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public
+License along with fiware-orion.
+If not, seehttp://www.gnu.org/licenses/.
+
+For those usages not covered by the GNU Affero General Public License
+please contact with::[iot_support@tid.es]
+"""
+
+__author__ = 'Jon Calderin Goñi (jcaldering@gmail.com)'
+
+# Created by Jon at 10/02/2015
+Feature: ServicePath for registrations (NGSI9) (not recursive)
+  # Enter feature description here
+  @iotcore-124
```

Comment:
It would be better to use github ticket number as reference, given that ITOCORE is not visible in public github, while github issue are.

All IOTCORE issue can be maped to the corresponding github issue in Orion, but in case of doubt, ask to me.